src/aat/resources/features/F-142 - Get Linked Case V2 External/S-142.6.td.json

```diff
@@ -0,0 +1,25 @@
+{
+  "_guid_": "S-142.6",
+  "_extends_": "Get_Linked_Cases__Base",
+  "title": "should retrieve linked case when the case reference exists",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains the case reference of the case just created",
+    "contains the details of the case just created, along with an HTTP-200 OK"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][childContexts][Standard_Full_Case_Creation_Data][testData][actualResponse][body][id]}"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "hasMoreRecords": false,
+      "linkedCases": [ ]
```

Comment:
Needs updating to include returned records

Reply:
Will be resolved in #1963 for [RDM-13140](https://tools.hmcts.net/jira/browse/RDM-13140) _"Functional Tests for the getLinkedCases API"_.